src/main/java/org/geomerty/oop/shapes/Triangle.java

```diff
@@ -0,0 +1,22 @@
+package org.geomerty.oop.shapes;
+
+public class Triangle {
+    private double a;
+    private double b;
+    private double c;
+
+    public Triangle(double a, double b, double c) {
+        this.a = a;
```

Comment:
@EhsanMaqs  specifying via 3 sides has drawback though. It's easy to specify sides that do not assemble in a triangle. e.g. `a=1, b=1,c=1000`. You might want to change it to other ways to specify the triangle or check the sides lengths and throw an exception if they do not make a triangle.